internal/configs/version2/nginx-plus.virtualserver.tmpl

```diff
@@ -1,6 +1,6 @@
 {{ range $u := .Upstreams }}
 upstream {{ $u.Name }} {
-    zone {{ $u.Name }} 256k;
+    {{if ne $u.UpstreamZoneSize "0"}}zone {{$u.Name}} {{$u.UpstreamZoneSize}};{{end}}
```

Comment:
the same problem as in the comment for the `nginx-plus.ingress.tmpl`.

Additionally, here there is a consistency problem. Compare the brackets usage `{{if ne $u.UpstreamZoneSize "0"}}zone` with the brackets usage in the surrounding lines like `{{ if $u.LBMethod }}`